### PR TITLE
Show more symbols on profiling builds

### PR DESCRIPTION
**Pull request by pwasowski2**
This change turns on more symbols in profiling view of DevTools and thus partially solves #95.

### CODE REVIEW

**Comment by swift-kim** [comment]
image

The artifact sizes are unacceptably large. Embedders shouldn't have dependencies on the dart runtime.

File: shell/platform/tizen/BUILD.gn [hunk 1]
```diff
@@ -161,6 +161,7 @@ template("embedder_for_profile") {
     public_deps = [ ":flutter_engine" ]
 
     deps = [
+      "//flutter/runtime:libdart",
```
Comment (bbrto21) [inline shell/platform/tizen/BUILD.gn:164]: Do we need this on all runtime modes?

**Comment by bbrto21** [comment]
@swift-kim Could you add ```-v``` option on Azure Pipelines?

File: shell/platform/tizen/BUILD.gn (hunk 1 above)
Comment (swift-kim) [reply shell/platform/tizen/BUILD.gn:164]: It's not required for all runtime mode unless `--unoptimized` is set.

Comment (swift-kim) [reply shell/platform/tizen/BUILD.gn:164]: It was not required even when `--unoptimized` is set in my case.

Comment (pwasowski2) [reply shell/platform/tizen/BUILD.gn]: I've turned it off on builds with `enable_profiling` set to `false`, to decrease the size of release builds.

File: shell/platform/tizen/BUILD.gn [hunk 2]
```diff
@@ -167,6 +170,10 @@ template("embedder_for_profile") {
       "//flutter/shell/platform/common/cpp/client_wrapper:client_wrapper",
       "//third_party/rapidjson",
     ]
+
+    if (enable_profiling) {
+      deps += [ "//flutter/runtime:libdart" ]
+    }
```
Comment (swift-kim) [inline shell/platform/tizen/BUILD.gn:176]: Why does the embedder depend on the dart runtime? Shouldn't you just remove these lines?

Comment (pwasowski2) [reply shell/platform/tizen/BUILD.gn]: Ok, I've removed the dependency completely

Comment (swift-kim) [reply shell/platform/tizen/BUILD.gn]: Thank you very much!

File: shell/platform/tizen/BUILD.gn [hunk 3]
```diff
@@ -5,6 +5,9 @@
 import("//flutter/shell/platform/common/cpp/client_wrapper/publish.gni")
 import("//flutter/shell/platform/tizen/config.gni")
 
+# define enable_profiling
+import("//build/config/profiler.gni")
+
```
Comment (swift-kim) [inline shell/platform/tizen/BUILD.gn:10]: You forgot to remove these lines.

Comment (pwasowski2) [reply shell/platform/tizen/BUILD.gn]: Oh,  sorry for that 🤦‍♂️ I've removed them now

**Review by swift-kim — APPROVED** [review]

**Review by bbrto21 — APPROVED** [review]
LGTM

**Pull request merged**